(DOCSP-32464): Updated List of Example Projects (#2968)
mongodb/docs-realm: source/example-projects.txt

Realm Example Projects
======================

> Explore engineering and community-provided example projects to learn best 

Reviewer: Since we have two table headings and one isn't above the fold, should we add an On this page TOC so the 2nd table isn't lost?
```
.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol
```

> Explore engineering and expert-provided example projects to learn best 

Reviewer: I think I used this sentence as an intro to the Kotlin project cards 😬 Might need to update there too (and any other pages that used it)

Author: Yes definitely!

practices and common development patterns for Realm and Atlas App Services.

> Foundational Projects

Reviewer: ```suggestion
Core Projects
```
Per our discussion, `Core` might be more understandable to users (especially ESL speakers) since it's often used in this context in development

---------------------

.. list-table::
  :header-rows: 1
  :widths: 25 50 25

  * - Project Name 
    - Description
    - Links for Available SDKs

  * - Connection State Change & Error Handling
    - Learn best practices around handling Sync errors and client reset strategies.

>     - - `Flutter <https://github.com/realm/realm-dart-samples/tree/kilochat/kilochat>`__

Reviewer: not sure if it really matters, but we do have a `:github:` role 
```suggestion
    - - :github:`Flutter <realm/realm-dart-samples/tree/kilochat/kilochat>`
```

      - `Kotlin <https://github.com/realm/realm-kotlin-samples/blob/main/AppServicesUsageSamples/apps/error-handling/README.md>`__
      - `.NET <https://github.com/realm/realm-dotnet/tree/main/examples/QuickJournal>`__
      - `Node.js <https://github.com/elle-j/realm-nodejs-example>`__
      - `Swift <https://github.com/realm/realm-swift-samples/blob/main/SyncConnectionStateErrorReset/README.md>`__

  * - Offline Login and Realm Access
    - Log in a Device Sync user and open a synced realm offline.
    - - `Flutter <https://github.com/realm/realm-dart-samples/tree/kilochat/kilochat>`__
      - `Kotlin <https://github.com/realm/realm-kotlin-samples/blob/main/AppServicesUsageSamples/apps/property-encryption/README.md>`__
      - `.NET <https://github.com/realm/realm-dotnet/tree/main/examples/QuickJournal>`__
      - `Swift <https://github.com/realm/realm-swift-samples/tree/main/OfflineLoginRealmAccess>`__

  * - Template Apps
    - A todo list mobile app that syncs data with App Services using Device Sync. See

>       the App Services `Getting Started page <https://www.mongodb.com/docs/atlas/app-services/get-started/>`__ for more information 

Reviewer: so it doesn't break if that URL changes
```suggestion
      the App Services :ref:`Getting Started page <app-services-get-started>` for more information 
```
